examples/bvh_driver/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@
 # wrong and thus, we cannot check the version during the configuration step.
 find_package(benchmark REQUIRED)
 
+find_package (Threads REQUIRED)
```

Comment:
I needed this for a non-`MPI` build.

Reply:
I think this is something about Google benchmark not recording properly their dependency:
https://github.com/google/benchmark/blob/7d856b03047e7bf09ccc9e9878e299493541b468/CMakeLists.txt#L258-L260
and
https://github.com/google/benchmark/blob/7d856b03047e7bf09ccc9e9878e299493541b468/src/CMakeLists.txt#L30-L41